.github/workflows/master_push.yml

```diff
@@ -72,8 +72,6 @@ jobs:
         run: chmod +x gradlew
       - name: Build and push GCR image using Jib
         run: "./gradlew jib --image=gcr.io/${GOOGLE_PROJECT}/${SERVICE_NAME}:${{ steps.tag.outputs.tag }}"
-        env:
-          SERVICE_VERSION: ${{ steps.tag.outputs.tag }}
```

Comment:
I think you have the merge from master messed up somehow. Let's make sure we don't revert the cloud profiler changes here and in build.gradle.